### ISSUE TITLE
How about Training?

### ISSUE THREAD

**Issue by Adeel-Intizar** [Licht-T/tf-centernet#1]
How can i train it on my custom dataset

**Comment by Licht-T** [comment]
Unfortunately, the training code is not implemented yet.

**Comment by Adeel-Intizar** [comment]
Do you intend to implement it?

**Comment by Licht-T** [comment]
Yes.

I am currently working on implementing DLA-34 backbone since Hourglass-104 backbone is too slow. DLA-34 utlizes deformable convolutions. Tensorflow doesn't have such functuon, so I made the PR.
https://github.com/tensorflow/addons/pull/2196

After this PR merged and DLA-34 implementation finished, I will make the training code.

**Comment by Adeel-Intizar** [comment]
I hope you implement it soon, looking forward to that

**Comment by Adeel-Intizar** [comment]
@Licht-T And how about extending this implementation to instance segmentation like CenterMask?

**Comment by Adeel-Intizar** [comment]
Do the weights file also generate masks for coco classes?

**Comment by Adeel-Intizar** [comment]
@Licht-T Pardon me for asking too many questions here in advance,
If you are busy these days and can't implement the training function, I can implement it but you have to provide the details (e.g. what data format you used in generator for training and validation etc.) of this implementation if that's okay for you.
